### PR TITLE
Html forms

### PR DESCRIPTION
**Pull request by Habib-Urr-Rehman**
Please Review

### CODE REVIEW

File: HTML FORMS/home.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home Page</title>
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <!-- Custom CSS -->
+    <style>
```
Comment (hussain021) [inline HTML FORMS/home.html]: Please create a separate file for style.

File: HTML FORMS/home.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    
+</div>
+
+<!-- Bootstrap Bundle JS -->
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+    crossorigin="anonymous"></script>
+
+<!-- JavaScript for logout -->
+<script>
```
Comment (hussain021) [inline HTML FORMS/home.html]: Please create a separate file for script.

File: HTML FORMS/home.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                My name is Habib. I am currently pursuing my BS in Computer Science degree from UET. As a dedicated computer science student, I am passionate about exploring the depths of technology and embracing the challenges that come with it. I am currently starting a new position at Arbisoft as an intern.
+            </p>
+          
+            <a href="https://drive.google.com/uc?export=download&id=1II7-TZFeRRClFDtXIvgrUJk7iVTVXqXV" class="btn btn-primary mt-3">Download CV</a>
+        </div>
+    </div>
+    
+</div>
+
+<!-- Bootstrap Bundle JS -->
```
Comment (hussain021) [inline HTML FORMS/home.html]: Please write this at the top of the file

File: HTML FORMS/login.html [hunk 4]
```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login</title>
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
```
Comment (hussain021) [inline HTML FORMS/login.html]: Same code written twice in codebase, you can use inheritance.

File: HTML FORMS/login.html [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+        <div class="col-md-4"></div>
+    </div>
+</div>
+
+<!-- Bootstrap Bundle JS -->
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+    crossorigin="anonymous"></script>
+
+<script>
```
Comment (hussain021) [inline HTML FORMS/login.html]: Move to script file

File: HTML FORMS/aa.jpg
Comment (hussain021) [inline HTML FORMS/aa.jpg:1]: This file should be name correctly.

**Review by hussain021 — CHANGES_REQUESTED** [review]
Please work on the requested changes at earliest convenience and fix all issues that are pointed out and look for the pointed out mistakes in the entire code (even if not commented upon)

**Comment by hussain021** [comment]
Please add replies on the comments so we can close this PR.

File: HTML FORMS/home.html (hunk 1 above)
Comment (Habib-Urr-Rehman) [reply HTML FORMS/home.html]: Yes, I created a separate file for styles. For Bootstrap-related CSS, I have added a separate file named `bootstrap.min.css`. For custom styles, I have included `styles.css`.

File: HTML FORMS/home.html (hunk 2 above)
Comment (Habib-Urr-Rehman) [reply HTML FORMS/home.html]: Yes, I created a separate file for scripts named `script.js`.

File: HTML FORMS/home.html (hunk 3 above)
Comment (Habib-Urr-Rehman) [reply HTML FORMS/home.html]: Yes , I have included this as the top of file

File: HTML FORMS/login.html (hunk 4 above)
Comment (Habib-Urr-Rehman) [reply HTML FORMS/login.html]: Yes, I utilize inheritance wherever feasible.

File: HTML FORMS/login.html (hunk 5 above)
Comment (Habib-Urr-Rehman) [reply HTML FORMS/login.html]: Yes, all types of scripts are now in separate files.

File: HTML FORMS/aa.jpg
Comment (Habib-Urr-Rehman) [reply HTML FORMS/aa.jpg:1]: Now this file is named correctly as Profile-Picture.jpg.

**Review by hussain021 — APPROVED** [review]
LGTM.

**Pull request merged**